### ISSUE TITLE
[Suggestion] Add ability to lock/unlock layout changes.

### ISSUE THREAD

**Issue by azeyad** [WebCabin/wcDocker#87]
Assume there is a scenario where changing the layout is allowed for the admin users only(or for any given user with a certain type of privilege) and in case the logged user is not authorized for such action I need a way to disable dragging panels around and changing the layout.

This can be even better if I can disable/enable layout changes at runtime after the page has been loaded,  but I am not sure about feasibility of that?

**Comment by ryanlin1986** [comment]
+1 for this.

**Comment by gbaumgart** [comment]
i think traversing over all panels and setting movable & collapsible to false should do the trick. You need a snippet for this ?

**Comment by azeyad** [comment]
@gbaumgart: I was thinking that having these properties on the main wcDocker window so that whenever we set them they apply on all child panels is more intuitive,  also after the page has been loaded, setting these properties at runtime should notify the child panels so they can set the movable & collapsible properties correspondingly.

**Comment by gbaumgart** [comment]
yeah, i understand but i think it would be little/unneeded bloat. I think its easier to subclass wcDocker and add two functions lock/unlock there (which could be called automatically after the options are parsed). I can post such code here if you like. Its really no biggy to do.

**Comment by Lochemage** [comment]
@azeyad 

I'm not against putting in some simple locking options that are docker wide such as disabling all panel movement. Does disabling panel re-sizing make sense as well (not sure if anyone would actually need that)?

**Comment by azeyad** [comment]
@Lochemage 
In my opinion we just need to disable panels dragging and closing functionalities, but not sure if there are other scenarios where disable re-sizing is going to be useful.
